### PR TITLE
available in other repositories

### PR DESCRIPTION
**Pull request by sadayuki-matsuno**
Changed to be available in other repositories.

Since rev-parse returns an error when there is no commit, it is changed to rev-list.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -41,9 +43,17 @@ func run() error {
 	flag.Parse()
 	now := time.Now().UTC()
 
+	repoOwner := os.Getenv("VULNLIST_REPOSITORY_OWNER")
```
Comment (knqyf263) [inline main.go]: It seems that LookupEnv is better. In addition, this process should be functionalized like utils.LookupEnv(key, defaultValue string).

File: git/git.go [hunk 2]
```diff
@@ -66,12 +66,16 @@ func clone(url, repoPath string) error {
...
-	revParseCmd := []string{"rev-parse", "HEAD"}
+	revParseCmd := []string{"rev-list", "-n", "1", "--all"}
 	output, err := utils.Exec("git", append(commandArgs, revParseCmd...))
 	if err != nil {
-		return nil, xerrors.Errorf("error in git rev-parse: %w", err)
+		return nil, xerrors.Errorf("error in git rev-list: %w", err)
 	}
 	commitHash := strings.TrimSpace(output)
+	if len(commitHash) == 0 {
+		log.Println("no commit yet")
```
Comment (knqyf263) [inline git/git.go]: Use logger package.

File: main.go (hunk 1 above)
Comment (sadayuki-matsuno) [reply main.go]: fixed

File: main.go [hunk 3]
```diff
@@ -41,10 +43,19 @@ func run() error {
...
+	if val, ok := os.LookupEnv("VULNLIST_REPOSITORY_NAME"); ok {
+		repoName = val
+	}
+
 	// Embed GitHub token to URL
 	githubToken := os.Getenv("GITHUB_TOKEN")
-	url := fmt.Sprintf(repoURL, githubToken)
+	url := fmt.Sprintf(repoURL, githubToken, repoOwner, repoName)
 
+	log.Printf("target repository is %s/%s", repoOwner, repoName)
```
Comment (knqyf263) [inline main.go]:
```suggestion
	log.Printf("target repository is github.com/%s/%s\n", repoOwner, repoName)
```

File: main.go (hunk 1 above)
Comment (knqyf263) [reply main.go]: Would you create new function such as `utils.LookupEnv(key, defaultValue string)`?

Comment (knqyf263) [reply main.go]: func LookupEnv(key, defaultValue string){
    if val, ok := os.LookupEnv(key); ok {
		return val
    }
    return defaultValue
}

**Pull request merged**